src/js/letters/containers/LetterList.jsx

```diff
@@ -66,7 +71,7 @@ export class LetterList extends React.Component {
     }
 
     return (
-      <div className="step-content">
+      <div className="step-content" aria-live="polite">
```

Comment:
I'm not super familiar with this--does the `aria-live` attribute propagate to this element's children? That is, the only things I can think of changing in here are the download buttons, but does this cover that, or do we need to make each button `aria-live="polite"`?

Reply:
In reading the docs on this it's hard to understand completely how this works, but my understanding is that this designates this element and its children as a live region, which means updates to that region will be read. I added this on `LetterList` so that when `CollapsiblePanel` is expanded, the content that describes each letter is read.